src/models/models.stack.js

```diff
@@ -453,7 +457,13 @@ export default class ModelsStack extends ModelsBase {
    */
   merge(stack) {
     // also make sure x/y/z cosines are a match!
-    if (this._stackID === stack.stackID) {
+    if (this._stackID === stack.stackID &&
+        this._numberOfFrames === 1 && stack._numberOfFrames === 1 &&
```

Comment:
`this._numberOfFrames === 1` is it needed?

Reply:
if `this` has several frames than `stack` shouldn't be merged to it